packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -454,13 +459,18 @@ function addEditableCallbacks<T extends DbObject>({collection, options = {}}: {
         }
         // FIXME: See comment on the other Connectors.create call in this file.
         // Missing _id and schemaVersion.
-        newRevisionId = await Connectors.create(Revisions, newRevision as DbRevision);
+        const newRevisionDoc = await createMutator({
+          collection: Revisions,
+          document: newRevision,
+          validate: false
+        });
+        newRevisionId = newRevisionDoc.data._id;
```

Comment:
The reason the new revision didn't have vote was because I was getting the `newRevisionId` wrong initially - I was doing `newRevisionDoc._id`